### PR TITLE
Updated XDEBUG documentation for drush debugging.

### PR DESCRIPTION
**Pull request by Nebel54**
I had a hard time getting xdebug working with drush, some things have changed since the documentation was written (e.g. using docksal/docksal-local.yml, drush.phar instead of a composer installation,...).

Not sure if i made a mistake when following the original documentation, but this was the only way for me to get xdebug running with drush.

### CODE REVIEW

**Comment by lmakarov** [comment]
Yeah, some of the more complex (and/or rarely used) use cases in our docs may still need updating.
Thanks for the PR! Going to review it soon.

File: docs/xdebug.md [hunk 1]
```diff
@@ -66,7 +62,25 @@ Replace **drupal7.docksal** with your domain. You can find it in `docksal.env` s
...
+
+- (1) You need to increase the Max. simultaneous connections to allow drush to spawn other drush instances. Otherwise the debugger might get stuck without any response.
+- (2) Disable "Force break at the first line when a script is outside the project". As the main drush binary resides in cli in /usr/local/bin/drush the debugger will break on every drush invocation.
+
+4) Install a site specific drush in your project root (if not already done) to get a non-phar version of drush.
+```
+fin exec composer require drush/drush:8.x
+```
+
+5.) Set path mappings for drush in PHP-Storm:
```
Comment (vgrygoryev) [inline docs/xdebug.md]: Please remove dot after 5.

**Comment by vgrygoryev** [comment]
It looks good for me. Thank you for this doc.
Only one thing:
"5.) Set path mappings for drush in PHP-Storm: "
I think we need to add note that this is optional. In most cases you don't need debug drush, you need debug only scripts in your project.

**Comment by Nebel54** [comment]
Great, fixed the typo and added a hint, that only steps 1 and 2 are needed for simple scripts.

File: docs/xdebug.md [hunk 2]
```diff
@@ -17,14 +17,10 @@ You can also pick from the [list](https://confluence.jetbrains.com/display/PhpSt
 
 ## Setup
 
-1) Set environment variable on the `cli` service
+1) Set environment variable on the `cli` in .docksal/dockasl.env
 
 ```yaml
```
Comment (lmakarov) [inline docs/xdebug.md]: Change this to "bash" as this is not longer "yaml"

**Comment by Nebel54** [comment]
@lmakarov I added the changes you requested.

**Comment by lmakarov** [comment]
Thanks. I added some final touches to get this merged quickly.

**Pull request merged**